### DISCUSSION TITLE
Classroom GPT

### DISCUSSION THREAD

**Discussion by FakeWood** [1101-GS4719-C/Python-Community-GS4719-C#1672] [General]
[NCUEDU](https://ncuedu.tw/)上的Classroom GPT可以讓學生問問題並即時獲得解答，

還能讓老師根據同學的提問來了解學生的學習情況。

不過關於旁邊的 ***同學們都問什麼*** 欄位，根據觀察，除了新加進去的同學看不到之前的紀錄以外，

如果切到其他瀏覽器標籤或是視窗的話，那在切換期間的資訊也不會更新，沒辦法看到在這段時間內其他同學問的問題，

我覺得這是未來可以改進的一個地方~

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105701699/ccf9bd50-cad0-4a3c-bae4-9f4d873b308a)

**Comment by 1chooo** [comment]
謝謝同學的建議！

這確實是個非常好的建議，可以讓歷史紀錄給大家查看，不過可能也有一些現實層面的考量，像是效能，如果今天太多同學一次發問，想必從伺服器撈資料的時間會變得很長，甚至可能導致仔入歷史對話紀錄太長而造成同學使用 Classroom GPT 使用效果不佳也是有可能。

不過確實是未來可以嘗試的一大努力目標沒錯！

**Reply by blacksai2019** [reply]
hi
